tests/data/hocr_simple/md.hocr

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This could just be a comment in this PR, but it might be nice to note in the PR how these test files were generated just for future reference.

Like the previously existing ones were written in markdown then used pandoc to generate PDF and HTML.

Reply:
Totally agreed!

Regarding md.hocr, 
1. Rasterize the `tests/data/pdf_simple/md.pdf` in gray
2. Apply OCR (Tesseract 4.1.1)
More specifically, here is what I did:
```
$ pdftoppm -gray -r 300 md.pdf md
$ tesseract --dpi 300 --psm 12 md-1.pgm md hocr
```

Regarding japan.hocr,
1. Simplied `tests/data/pure_html/japan.html` and saved it in HTML (pasted below).
2. Print the simplified HTML using Google Chrome in PDF.
3. Follow the same steps as md.hocr

```html
<!DOCTYPE html>
<html class="client-nojs" lang="ja" dir="ltr">

<head>
    <title>ジャパン - Wikipedia</title>
</head>

<body class="mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject page-ジャパン rootpage-ジャパン skin-vector action-view">
<p>「にっぽん」あるいはその異読である「じっぽん」の転訛。当時マルコ・ポーロが辿り着いたと言われる元の言語で「日本国」のことをそのような発音で呼んでいた。</p>

</body>
</html>
```

Reply:
Regarding 121.hocr	and 1st.hocr, they were also generated from PDFs.
The dataset is available to download at https://www.consumerfinance.gov/credit-cards/agreements/.
1. Download the Archived Q4-2019 agreements
2. Unzip it and find 
    - "121 Financial Credit Union_Visa Credit Card Agreement.pdf" (for 121.hocr)
    - "1st Advantage Federal Credit Union_Platinum Visa Credit Card Agreement.pdf" (for 1st)
3. For 121.hocr, did the same steps as "md.hocr".

"1st.hocr" is a bit complicated and not really reproducible.
I used pdftotext to extract the text part and used a proprietary tool to recognize tables.
Nevertheless, "1st.hocr" serves just an example hocr that contains tables, so it is easily replaceable with something else.

Furthermore, I edited both hocr to make it minimal but good enough to demonstrate problems that I encountered when I was dealing with real hOCRs generated using Tesseract. These problems include
- Multi-to-one and one-to-multi token mappings
- A single ocrx_word could contain two words
- CJK languages that does not have whitespaces between words

Hope this description serves as a good future reference.